README.md

```diff
@@ -8,9 +8,16 @@ If you are using Windows, we recommend downloading Windows Subsystem for Linux,
 
 ## Linux Prerequisites
 FOUND runs on a Linux System that has many requirements. The following packages are required, with download instructions given in Ubuntu:
+- Git (`sudo apt install git`)
```

Comment:
I would like to but then also mac? So I feel it might be best to just leave this for people to do themselves. I was thinking of putting this in the makefile, but then I'd run into the same issue.